apps/storefront/src/utils/getCartUrl.ts

```diff
@@ -0,0 +1,8 @@
+export default function getCartUrl() {
+  const {
+    setting: { platform, cart_url: cartUrl },
+  } = window.B3;
+  const fallbackValue = platform === 'bigcommerce' ? '/cart.php' : '/cart';
+
+  return cartUrl || fallbackValue;
```

Comment:
@bc-victor 

can we move this logic to the `src/constants/index.ts` file?

and replace the `CART_URL` value there? That should allow you to not need to change all the places it is currently used.